### DISCUSSION TITLE
Errore con certificato HTTPS

### DISCUSSION THREAD

**Discussion by mippo13** [globaleaks/globaleaks-whistleblowing-software#3885] [Q&A]
Buongiorno, a causa di un problema tecnico, abbiamo dovuto reinstallare la piattaforma. 
Il dominio https, registrato correttamente nella prima installazione (whistleblowing.xxx.it), ora non è più rinnovabile (la piattaforma restituisce un errore interno). Non abbiamo purtroppo un backup della prima piattaforma installata.
Esiste un modo per riavere il nome di dominio della prima installazione? O dobbiamo attendere la sua naturale scadenza ( mi pare fosse a marzo del 2024)?
Grazie in anticipo della risposta.
Cordiali saluti.

**Comment by evilaliv3** [comment]
Perchè i certificati let'sencrypt possano essere rinnovati è necessario che la piattaforma sia pubblicamente raggiungibile suller porte 80 e 443.
E' possibile che la porta 80 sia bloccata da un firewall?

**Comment by mippo13** [comment]
Si certo. Temo che il problema sia che il certificato è stato emesso con la prima installazione che purtroppo non ho potuto recuperare. La seconda installazione, quando richiede il medesimo certificato da un errore, probabilmente perchè già emesso e non revocato.

**Comment by ggiomma** [comment]
Buongiorno, stessa situazione. Sito offline per molto tempo, nuova installazione della versione 4.13.18 su Debian 12 ed errore su rinnovo certificato; aggiornato oggi alla 4.13.22 ma il risultato non cambia.

**Comment by evilaliv3** [comment]
Gli errori riportati sono piuttosto insoliti e credo differenti.

Se potete fornire un link alle vostre piattaforme possiamo provare ad
assistervi?

**Reply by mippo13** [reply]
Buongiorno, io ho risolto registrando un altro nome (wb.XXXX.it invece di whistleblowing.XXXX.it). La cosa non mi piace ma ahimè oramai va così (il tutto è stato approvato dagli organi interni e quindi non si cambia).
Ma per approfondire e capire bene la questione, appena ho un momento, creo una nuova macchina/installazione e provo...e vi mando il link.
Saluti.

**Reply by ggiomma** [reply]
Nel nostro caso, il sito è rimasto offline per molto tempo e nel frattempo è cambiato l'ip pubblico (altro provider; abbiamo il sito su una macchina virtuale all'interno della nostra rete locale); probabilmente l'ip registrato da Let'sencript è ancora quello vecchio.
Ad ogni modo, il nostro sito è: segnalazioni.commerfidi.it
Grazie

**Reply by spomar** [reply]
il certificato che state usando sul webserver segnalazioni.commerfidi.it e' associato al dominio localhost
E' stato generato in maniera totalmente errata
Dovete rigenerare il cerificato assegnadogli segnalazioni.commerfidi.it e potrete usarlo su qualsiasi webserver
